### DISCUSSION TITLE
Full details of my octernship project

### DISCUSSION THREAD

**Discussion by abhay-raj19** [community/community#50122] [GitHub Education]
I have applied for the octernship, but one of the projects automatically assigned by github classroom bot does not even contain a readme file so I am unable to go about that project what should I d to fix this. Am i able to withdraw from that project and apply for a new one?
it is the to-do-list with appwire and flutter.
From where would i will get the full details as what  i have to do  and many more things related to the projects.

**Comment by abhay-raj19** [comment]
It would be great help for me if someone solves up my problem.

Waiting for the answers.

**Reply by saaxena** [reply]
Go through the repositories of organisation you have applied , they had given the  tasks details  or if it doesn't work contact the project team .
Hope it helps 🙂
